peer/brontide.go

```diff
@@ -424,6 +424,10 @@ type Brontide struct {
 	// the state machine will be deleted from the map.
 	activeChanCloses map[lnwire.ChannelID]*chancloser.ChanCloser
 
+	// activeChanClosesMtx is a mutex that guards access to the
+	// activeChanCloses map.
+	activeChanClosesMtx sync.RWMutex
```

Comment:
Is this mutex needed given (afact), this is always accessed from a single event loop.